.github/workflows/scripts/test-env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Could: This is better portable

```suggestion
#!/usr/bin/env bash
```